app/assets/src/components/views/PipelineViz.jsx

```diff
@@ -321,37 +529,48 @@ class PipelineViz extends React.Component {
     graph.moveNodeToPosition(END_NODE_ID, xEndNodePos, yStartNodePos);
   }
 
-  closeNonativeSteps() {
-    this.stagesData.forEach((stageData, i) => {
-      const graph = this.graphs[i];
-      if (stageData.jobStatus !== "STARTED") {
-        graph.afterDrawingOnce(() => {
-          this.toggleStage(i);
-        });
-      }
-    });
+  closeIfNonativeStage(graph, stageIndex) {
```

Comment:
Did you mean `Nonactive` ?
nit: also reads better if you capitalize the a, i.e. `...NonActive...`